### PR TITLE
chore: improve share wallet test

### PR DESCRIPTION
**Pull request by J-Son89**
test needed some adjustment to work properly with async behaviours

### CODE REVIEW

**Comment by eidoo511** [comment]
Is there a solution to take a copy of iCloud that was deleted from Apple?

File: src/status_im/contexts/shell/share/wallet/component_spec.cljs [hunk 1]
```diff
@@ -26,19 +26,24 @@
...
         (.then (fn []
                  (h/fire-event :press (h/get-by-label-text :share-qr-code-multichain-tab))
-                 (-> (h/wait-for #(h/is-truthy (h/query-by-text "eth:")))))))))
-
+                 (-> (h/wait-for #(h/get-by-label-text :share-qr-code-settings))
+                     (.then (fn []
+                              (h/is-truthy (h/get-by-text "eth:"))
+                              (h/is-truthy (h/get-by-text "opt:"))
+                              (h/is-truthy (h/get-by-text "arb1:"))
+                            ))))))))
```
Comment (ilmotta) [inline src/status_im/contexts/shell/share/wallet/component_spec.cljs]: Quick one: this line should be joined with the previous one.

**Review by ilmotta — APPROVED** [review]

**Review by mohsen-ghafouri — APPROVED** [review]

**Comment by J-Son89** [comment]
@status-im/mobile-qa - this just updates a component spec test so no manual QA needed for this pr. It also should not effect e2e tests in any way either. I'll run them anyway but provided `jenkins/prs/tests` passes this pr should be safe to merge 👍

**Comment by pavloburykh** [comment]
> @status-im/mobile-qa - this just updates a component spec test so no manual QA needed for this pr. It also should not effect e2e tests in any way either. I'll run them anyway but provided `jenkins/prs/tests` passes this pr should be safe to merge 👍

@J-Son89 thank you! I believe in this case it is okay to skip e2e step. So PR is ready for merge.

**Pull request merged**